docs/cody/clients/enable-cody-enterprise.mdx

```diff
@@ -79,7 +79,36 @@ To turn Cody off:
 
 ## Enable Cody only for some users
 
-To enable Cody only for some users, for example, when rolling out a Cody POC, follow all the steps mentioned in [Enabling Cody on your Sourcegraph instance](#enable-cody-on-your-sourcegraph-instance). Then, use the feature flag `cody` to turn Cody on selectively for some users. To do so:
+To enable Cody only for some users, for example, when rolling out a Cody POC, follow all the steps mentioned in [Enabling Cody on your Sourcegraph instance](#enable-cody-on-your-sourcegraph-instance). Then, do the following:
+
+<details>
```

Comment:
```suggestion

```
These HTML tags will not work. We will soon get a `toggle` component that will make it easier. Till then we can use an H3. :)